.gitignore

```diff
@@ -12,5 +12,7 @@ MANIFEST
 /.coverage
 /htmlcov/
 /env/
+*.sublime-project
```

Comment:
Lines like this belong in your personal git `core.excludes` file instead of being checked in.